index.js

```diff
@@ -2,12 +2,19 @@ import express from 'express';
 import {config} from "./config/config.js";
 import controller from './delivery/controller.js'
 import errorHandler from './exception/error_middleware.js';
+import nunjucks from 'nunjucks';
 
 const app = express();
 
 app.use(controller)
 app.use(errorHandler)
 
+nunjucks.configure('templates', {
+    autoescape: true,
+    express: app
+})
```

Comment:
this is not a mistake, but it is better to always maintain a certain order in the sequence of calls - middlewares, routes, error handlers, running server

```js
const app = express();

nunjucks.configure('templates', {
    autoescape: true,
    express: app
})
app.use(controller)
app.use(errorHandler)

app.listen()..
```